fil-proofs-tooling/src/bin/benchy/window_post.rs

```diff
@@ -603,9 +597,14 @@ pub fn run(
     skip_commit_phase1: bool,
     skip_commit_phase2: bool,
     test_resume: bool,
-    use_synthetic: bool,
+    api_features: Vec<ApiFeature>,
 ) -> anyhow::Result<()> {
-    info!("Benchy Window PoSt: sector-size={}, api_version={}, preserve_cache={}, skip_precommit_phase1={}, skip_precommit_phase2={}, skip_commit_phase1={}, skip_commit_phase2={}, test_resume={}, use_synthetic={}", sector_size, api_version, preserve_cache, skip_precommit_phase1, skip_precommit_phase2, skip_commit_phase1, skip_commit_phase2, test_resume, use_synthetic);
+    let api_features_str = api_features
+        .iter()
+        .map(|api_feature| api_feature.to_string())
+        .collect::<Vec<String>>()
+        .join(",");
+    info!("Benchy Window PoSt: sector-size={}, api_version={}, preserve_cache={}, skip_precommit_phase1={}, skip_precommit_phase2={}, skip_commit_phase1={}, skip_commit_phase2={}, test_resume={}, apu_features={}", sector_size, api_version, preserve_cache, skip_precommit_phase1, skip_precommit_phase2, skip_commit_phase1, skip_commit_phase2, test_resume, api_features_str);
```

Comment:
Typo: `apu_features` should be `api_features`, but I think `features` is plenty good for debug/logging info